MatrixSDK/Crypto/Data/MXDeviceListOperationsPool.m

```diff
@@ -169,8 +171,12 @@ - (void)doKeyDownloadForUsers:(NSArray<NSString *> *)users token:(NSString *)tok
                 // Update the store
                 // Note that devices which aren't in the response will be removed from the store
                 [self->crypto.store storeDevicesForUser:userId devices:mutabledevices];
+                
+                usersDevices[userId] = mutabledevices.allValues;
             }
         }
+        
+        [[NSNotificationCenter defaultCenter] postNotificationName:MXDeviceListDidUpdateUsersDevicesNotification object:nil userInfo:usersDevices];
```

Comment:
We need to post the notification from a defined instance. Else, in case of multi-account, we are going to be completely lost.
Unfortunately, we cannot use `self` because this class is private. The best candidate is probably `self->crypto`  but we need to move the notification definition to `MXCrypto`.